assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -1497,7 +1497,11 @@ public String getSubmissionStatus(String submissionId) {
     public int getSubmittedSubmissionsCount(String assignmentReference) {
```

Comment:
getSubmittedSubmissionsCount and getUngradedSubmissionsCount could be collapsed into a single method.

Reply:
@ern 

In the service? Really? I think it's better to have only one method in the persistence and have two methods in the service, if not I'll need to put again logic in the UI to ask for the ungraded/submitted value to the service.

What do you think?

Reply:
I've collapsed it in one method controlled by a boolean from the UI.